src/scripts/main.js

```diff
@@ -1 +1,93 @@
 'use strict';
```

Comment:
I wont say anything about your scripts just 'cuz  you know it better than me)) :sweat_smile: